config/routes.rb

```diff
@@ -38,6 +38,8 @@
 
     mount BopsReports::Engine, at: "/reports", as: :bops_reports
 
+    mount BopsSubmissions::Engine, at: "/submissions", as: :bops_submissions
```

Comment:
Might want to mount this at `/` so we can do:

``` ruby
resources :submissions, only: %i[create update]
```

in the engine's routes file